application/auth/models.py

```diff
@@ -28,8 +29,8 @@ class TypeOfUser(enum.Enum):
 
 DEPT = [READ, UPDATE_MEASURE, CREATE_VERSION]
 RDU = DEPT + [CREATE_MEASURE, DELETE_MEASURE, VIEW_DASHBOARDS]
-ADMIN = RDU + [PUBLISH, MANAGE_USERS, ORDER_MEASURES]
-DEV = ADMIN + [COPY_MEASURE, MANAGE_SYSTEM]
+ADMIN = RDU + [PUBLISH, MANAGE_USERS, ORDER_MEASURES, MANAGE_DATA_SOURCES]
+DEV = ADMIN + [COPY_MEASURE, MANAGE_SYSTEM, MANAGE_DATA_SOURCES]
```

Comment:
You'll need to include a data migration (and manually run it after release) that will add these capabilities to all the affected users, as these are only used to populate capabilities for users upon account creation - they are then just stored as a static, duplicated array of strings representing permissions for each user.

Reply:
Good point - will add.

Reply:
Added in 2595e08e01239ee8ab0ff0e2850ba5e4bc20c31e